assessments/README.md

```diff
@@ -35,24 +34,20 @@ the assessements.
 
 ## Outcome
 
-Each project assessment will:
+Each CNCF project security assessment will:
 1. ensure a clear description of the project's design goals with respect to
 security
-2. uncover design flaws and document known limitations
+2. uncover design and configuration flaws while documenting known limitations
 3. document next steps toward increasing security of the project itself and/or
-increasing the applications of the project toward increasing security of the
-cloud native ecosystem
+increasing the applications of the project toward a more secure cloud native ecosystem
 
 Due to the nature and timeframe for the analysis, *this review is not meant
 to subsume the need for a professional security audit of the code*.  Audits
-of implementation vulnerabilities and similar issues at that level are not
-intended to be covered by this assessment.  The purpose of this effort is to
-uncover design flaws and to obtain a clear articulation of what the project's
-design goals and security properties are intended to be.
+of implementation specific vulnerabilities, improper deployment configuration, etc. are not in scope of a CNCF project security assessment.  A CNCF project security assessmet is intended to uncover design and configuration flaws and to obtain a clear, comprehensive articulation of the project's design goals and aspirations while documenting the intended security properties enforced, fulfilled, or executed by said project.
```

Comment:
"implementation specific" => "implementation-specific"

Also, I think wrapping 80 columns makes the diffs easier to read on github and seems to be what folks are doing in the rest of the repo

Reply:
changed